diff --git a/dist/providers/github/index.js b/dist/providers/github/index.js
index d977fdb305703b584a5d8df1dd99dabf0afaf7fd..6db20d24afa9f2cf6b928d0d390aedd4ca958c50 100644
--- a/dist/providers/github/index.js
+++ b/dist/providers/github/index.js
@@ -145,7 +145,9 @@ var AuthFlow = class {
     if ("message" in response) {
       throw new import_http_exception.HTTPException(400, { message: response.message });
     }
-    response.email = await this.getEmail();
+    if (!this.oauthApp) {
+      response.email = await this.getEmail();
+    }
     if ("id" in response) {
       this.user = response;
     }
diff --git a/dist/providers/github/index.mjs b/dist/providers/github/index.mjs
index 628a582ef67d62d9b2856993726738523bdeb333..ab599f24f54f79b0303c487d282ae845124c235f 100644
--- a/dist/providers/github/index.mjs
+++ b/dist/providers/github/index.mjs
@@ -119,7 +119,9 @@ var AuthFlow = class {
     if ("message" in response) {
       throw new HTTPException(400, { message: response.message });
     }
-    response.email = await this.getEmail();
+    if (!this.oauthApp) {
+      response.email = await this.getEmail();
+    }
     if ("id" in response) {
       this.user = response;
     }
